lib/rules/order-in-components.js

```diff
@@ -32,7 +33,9 @@ module.exports = {
 
   create(context) {
     const options = context.options[0] || {};
-    const order = options.order || ORDER;
+    let order = options.order || ORDER;
+    order = addBackwardsPosition(order, 'empty-method', 'method');
```

Comment:
This is a tiny bit confusing to me. If no order is provided through the options - and we use the `ORDER` const - there is no need to call `addBackwardsPosition` on it, since we have `empty-method` already defined where it should be.
I believe that for increased clarity in the code, we should only call `addBackwardsPosition` when actually necessary, for example:
```
const order = options.order ? addBackwardsPosition(options.order, 'empty-method', 'method') : ORDER;
```